ce/api/metadata.py

```diff
@@ -50,23 +50,22 @@ def metadata(sesh, model_id):
                     'timescale': 'monthly',
                     'times':
                     {
-                        0: '1985-01-15T00:00:00Z',
-                        1: '1985-02-15T00:00:00Z',
-                        2: '1985-03-15T00:00:00Z',
-                        3: '1985-04-15T00:00:00Z'
+                        0: datetime.datetime(1985, 1, 15, 0, 0),
+                        1: datetime.datetime(1985, 2, 15, 0, 0),
+                        2: datetime.datetime(1985, 3, 15, 0, 0),
+                        3: datetime.datetime(1985, 4, 15, 0, 0),
                     }
                     'multi_year_mean': False,
-                    'start_date': '1985-01-15T00:00:00Z',
-                    'end_date': '1985-04-15T00:00:00Z',
+                    'start_date': 'datetime.datetime(1985, 1, 15, 0, 0),
+                    'end_date': datetime.datetime(1985, 4, 15, 0, 0),
+                    'modtime': datetime.datetime(2010, 1, 1, 17, 30, 4)
```

Comment:
I really approve of keeping datetimes in native representation until final conversion.